src/Wallabag/CoreBundle/Resources/views/base.html.twig

```diff
@@ -70,10 +70,6 @@
 
         {% block footer %}{% endblock %}
 
-        <div id="warning_message">
```

Comment:
This can be moved to the config_prod.yml file and should be disabled in the config_dev.yml.
I don’t like this message in dev :)

And yep it was in the parameters.yml before, but I don’t think it should be a param that must be defined by the end user when it install wallabag.


On 3 avril 2016 at 20:31:17, Thomas Citharel (notifications@github.com) wrote:

In src/Wallabag/CoreBundle/Resources/views/base.html.twig:

> @@ -70,10 +70,6 @@
>   
>          {% block footer %}{% endblock %}
>   
> -        <div id="warning_message">
That's what I thought. How come we changed it ?

—
You are receiving this because you were assigned.
Reply to this email directly or view it on GitHub